Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -511,124 +543,38 @@ export default function AssetPage(props: IProps) {
-                                                        asset.Channels = _.uniqBy(c.map(ch => ({ ...ch, ConnectionPriority: 1 }))
-                                                            .concat(newEditAsset.Channels.filter(ch => ch.ConnectionPriority != 1)), (ch) => ch.ID);
-                                                        setNewEditAsset(asset);
-
-                                                        //Update Channels with new ConnectionPriority
-                                                        let channels = _.clone(props.Channels);
-                                                        channels = channels.map(ch => ({ ...ch, ConnectionPriority: c.find(d => d.ID == ch.ID) == null ? ch.ConnectionPriority : 1 }));
-                                                        props.UpdateChannels(channels);
-                                                    }}
-                                                />
-                                            </div>
-                                            <div className="col-4 d-flex flex-column">
-                                                <ChannelSelector
-                                                    Label="Associated Channels Tertiary Side"
-                                                    Channels={props.Channels}
-                                                    SelectedChannels={newEditAsset.Channels.filter(ch => ch.ConnectionPriority == 2)}
-                                                    UpdateChannels={(c) => {
-                                                        let asset = _.clone(newEditAsset as OpenXDA.Types.Asset);
-                                                        asset.Channels = _.uniqBy(c.map(ch => ({ ...ch, ConnectionPriority: 2 }))
-                                                            .concat(newEditAsset.Channels.filter(ch => ch.ConnectionPriority != 2)), (ch) => ch.ID);
-                                                        setNewEditAsset(asset);
-
-                                                        //Update Channels with new ConnectionPriority
-                                                        let channels = _.clone(props.Channels);
-                                                        channels = channels.map(ch => ({ ...ch, ConnectionPriority: c.find(d => d.ID == ch.ID) == null ? ch.ConnectionPriority : 2}));
-                                                        props.UpdateChannels(channels);
-                                                    }}
-                                                />                                                
-                                            </div>
-                                        </div>
-                                    </div>
-                                ) : null}
+                            <div className="h-100" style={{paddingBottom: '6rem'}}> {/* kind of hacky to remove scroll bar b/c the select overflows */}
```

Comment:
you should be able to fix this. the `div` in 564 is taking up all space. 
inside you have a `TabSelector` which takes up whatever space it needs and whatever is inside `ChannelSelector`.
`ChannelSelector` contains an `input` which again takes up whatever space it needs and a `select` currently the `select is set to take `100%` of the appropriate parent. Instead use `Flex` (`display` on the parent and `flex: 1` etc on the apppropriate child)

use `flex` to make the `ChannelSelector take up the restt of the space.